server/routes/bitbucket/bitbucket_selection.py

```
@@ -345,6 +348,18 @@ def cleanup_org_hooks(user_id: str, org_id: str, repo_full_names=None) -> list[s
     return failed
 
 
+def _until_not_429(fn):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_until_not_429`

(ANN202)